monitoring/config-generator/src/generate.js

```diff
@@ -658,49 +658,37 @@ function getRules(allowList) {
 
   groups.push(broadcastingFunds)
 
-
-  let transcodingLatency = {
-    name: 'transcoding-latency',
-    rules: [{
-        alert: '50-pct-latency-high',
-        expr: 'histogram_quantile(0.5, sum(rate(livepeer_transcode_overall_latency_seconds_bucket[1m])) by ( le)) > 1',
-        for: '5m',
-        annotations: {
-          title: '50% transcoding latency is high',
-          description: 'Our 50% transcoding latency is greater than one second'
-        },
-        labels: {
-          severity: 'page'
-        }
-      },
+  let httpRealTimeRatio = {
+    name: 'http-real-time-ratio',
+    rules: [
       {
-        alert: '90-pct-latency-high',
-        expr: 'histogram_quantile(0.9, sum(rate(livepeer_transcode_overall_latency_seconds_bucket[1m])) by ( le)) > 2',
+        alert: '1x-real-time',
+        expr: '(sum(increase(livepeer_http_client_segment_transcoded_realtime_3x[1m])) + sum(increase(livepeer_http_client_segment_transcoded_realtime_2x[1m])) + sum(increase(livepeer_http_client_segment_transcoded_realtime_1x[1m]))) / sum(increase(livepeer_segment_source_emerged_unprocessed_total[1m])) < .99',
         for: '2m',
         annotations: {
-          title: '90% transcoding latency is high',
-          description: 'Our 90% transcoding latency is greater than two seconds'
+          title: '% 1x real-time HTTP push requests is high',
+          description: 'The % of HTTP push requests that complete in 1x real-time or faster is high'
```

Comment:
Is "faster" the right terminology here? The alarm goes off if it's "1x real-time or slower", yeah? Or are these descriptions written as the "success case"?